services/web/client/source/class/qxapp/auth/ui/LoginPage.js

```diff
@@ -90,16 +92,26 @@ qx.Class.define("qxapp.auth.ui.LoginPage", {
       this.add(grp);
 
       // TODO: add here loging with NIH and openID
-      // let grp2 = new qx.ui.container.Composite(new qx.ui.layout.HBox());
-      // ["Login with NIH", "Login with OpenID"].forEach(txt => {
-      //   let btn = this.createLinkButton(this.tr(txt), function(){}, this);
-      //   grp2.add(btn.set({
-      //     center: true
-      //   }), {
-      //     flex:1
-      //   });
-      // }, this);
-      // this.add(grp2);
+      // this.add(this.__buildExternals());
+    },
+
+    __buildExternals: function() {
+      let grp = new qx.ui.container.Composite(new qx.ui.layout.HBox());
+
+      ["Login with NIH", "Login with OpenID"].forEach(txt => {
+        let btn = this.createLinkButton(this.tr(txt), function() {
```

Comment:
this does not work as qx uses the this.tr structure to pick up the strings to translate which end up in the po files ... (translation itself would work with the variable but you would never know that you have to translate the Login with NIH string.)

Reply:
Thx! I knew about it ... but this is the evil of c&p in action :)